data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside, to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
+
+
+
+mission "Arachi Cartoonist 1"
+	minor
+	name "A New Design"
+	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs for his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
+	to offer
+		has "Coalition: First Contact: done"
+	source "Mebla's Portion"
+	destination "Prime"
+	on offer
+		conversation
+			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you on your tracks.`
+			`	He introduces himself as Drugub, a cartoonist who has seen some local fame lately for his sci-fi comics. "Many ship designs produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
+			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs, and settles for those.`
+			`	He thanks you and wishes you safe travels on your journey to <planet>.`
+				accept
+
+
+
+mission "Arachi Cartoonist 2"
+	name "A New Design"
+	description "Return to Drugub on <destination>, and give him the flyer with pictures and some details of Betelgeuse's ship models."
+	landing
+	to offer
+		has "Arachi Cartoonist 1: done"
+	source "Prime"
+	destination "Mebla's Portion"
+	on offer
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer, which you then unfold to see all of Betelgeuse's ships depicted, as well as some details on each (and the pricing for each too, of course).`
+			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`
+			`	You thank her, saying that your friend does not have the funds for any of the ships, and just wants some info, and head back to your ship.`
+			`	Hopefully a small pamphlet will do for Drugub to get some ideas.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 75940
+		conversation
+			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
```

Comment:
```suggestion
			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's a scroll full of ancient knowledge.`
```
Personal preference.